jenkins-scripts/ign_physics-default-devel-windows-amd64.bat

```diff
@@ -4,10 +4,8 @@ set VCS_DIRECTORY=gz-physics
 set PLATFORM_TO_BUILD=x86_amd64
 set IGN_CLEAN_WORKSPACE=true
 
-set DEPEN_PKGS=eigen3 tinyxml2
 for /f %%i in ('python "%SCRIPT_DIR%\tools\detect_cmake_major_version.py" "%WORKSPACE%\%VCS_DIRECTORY%\CMakeLists.txt"') do set GZ_MAJOR_VERSION=%%i

```

Comment:
Remove version checking logic since it's not needed anymore

Reply:
removed [a1452e9](https://github.com/gazebo-tooling/release-tools/pull/907/commits/a1452e9bcc3d4ccdce353dca99a4bfa0c1a75b2b)